services/api/src/resources/notification/resolvers.ts

```diff
@@ -705,3 +705,23 @@ export const removeAllNotificationsFromAllProjects: ResolverFn = async (
   // TODO: Check rows for success
   return 'success';
 };
+
+export const getAllNotifications: ResolverFn = async (
+  root,
+  args,
+  { sqlClientPool, hasPermission }
+) => {
+  await hasPermission('notification', 'viewAll');
+
+  const rows = await Helpers(sqlClientPool).selectAllNotifications();
+
+  const nids = await Helpers(sqlClientPool).selectAllProjectNotifications(rows);
```

Comment:
I'm not sure if it is a good idea to also return the project notifications here, as the return data will be overly complicated

Reply:
There would be a lot of duplicates and the only difference would be the `pid` field

Reply:
The project notifications shouldn't be returned. Only iterating over them to assign the project id to what were getting in `rows`, which is then returned.

Reply:
Yeah, I think strip it out, because a notification can be attached to 1 or more projects, so the `pid` would only ever be 1 of those projects